src/blockUser.js

```diff
@@ -0,0 +1,37 @@
+"use strict";
+
+var utils = require("../utils");
+var log = require("npmlog");
+
+module.exports = function(defaultFuncs, api, ctx) {
+  return function blockUser(userID, callback) {
+
+    if (!callback) {
+      callback = function() {};
+    }
+
+    defaultFuncs
+      .post("https://www.facebook.com/nfx/block_messages/?thread_fbid=" + userID + "&location=www_chat_head", ctx.jar, {})
+      .then(utils.saveCookies(ctx.jar))
+      .then(utils.parseAndCheckLogin(ctx.jar, defaultFuncs))
+      .then(function(resData) {
+        if (resData.error) {
+          throw resData;
+        }
+        defaultFuncs
+          .post("https://www.facebook.com" + /action="(.+?)"+?/.exec(resData.jsmods.markup[0][1].__html)[1].replace(/&amp;/g, "&"), ctx.jar, {})

```

Comment:
```
function get(url, jar, qs) {
  // I'm still confused about this
  if(getType(qs) === "Object") {
    for(var prop in qs) {
      if(qs.hasOwnProperty(prop) && getType(qs[prop]) === "Object") {
        qs[prop] = JSON.stringify(qs[prop]);
      }
    }
  }
  var op = {
   ...
    url: url,
    method: "GET",
...
```

Looking at [utils.js#L27](../blob/master/utils.js#L27) (the only _qs_ variable I found), that is for a GET request. I was using a POST, since Chrome's developer tools said that it was so